packages/webview_flutter/webview_flutter/lib/src/webview_controller.dart

```diff
@@ -43,16 +43,30 @@ import 'webview_widget.dart';
 class WebViewController {
   /// Constructs a [WebViewController].
   ///
+  /// `onPermissionRequest`: Sets a callback that notifies the host application
+  /// that web content is requesting permission to access the specified
+  /// resources. Apps that attempt to access device resources will most likely
+  /// require additional platform=specific steps for acquiring permission from
+  /// the user.
```

Comment:
I'm not sure I follow this last sentence. Is it just trying to say that the app needs the relevant system-level permission, if applicable, in order for granting it to the web view to work? If so, I think we should say that more clearly, and give an illustrative example (e.g., how camera permissions would play out on Android and iOS).